prep-javascript-objects/main.js

```diff
@@ -0,0 +1,13 @@
+var person = {
+  firstName: "Jeesu",
+  lastName: "Park",
+  hobby: "basketball",
+};
+console.log(person);
+var firstName = person.firstName
```

Comment:
This should be `fullName` so both the first and last name together with a space between the names.

Reply:
oh I definitely messed that one up.. 
thank you haha